### PR TITLE
Update learn rounding digits to handle all nan data

### PR DESCRIPTION
**Pull request by katxiao**
Resolves https://github.com/sdv-dev/SDV/issues/574

### CODE REVIEW

File: rdt/transformers/numerical.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def _learn_rounding_digits(data):
                 if (roundable_data == roundable_data.round(decimal)).all():
                     return decimal
 
-        else:
+        elif len(roundable_data) > 0:
```
Comment (amontanez24) [inline rdt/transformers/numerical.py:85]: I assume `elif roundable_data` doesn't work because it still creates a series with no rows right?

**Review by amontanez24 — APPROVED** [review]
LGTM :shipit:

File: rdt/transformers/numerical.py (hunk 1 above)
Comment (katxiao) [reply rdt/transformers/numerical.py:85]: Yeah it's an empty series

**Review by csala — APPROVED** [review]
Looks good!

**Pull request merged**